include/deal.II/distributed/tria_base.h

```diff
@@ -97,7 +97,7 @@ namespace parallel
      * Return MPI communicator used by this triangulation.
      */
     virtual MPI_Comm
-    get_communicator() const override;
+    get_mpi_communicator() const override;
```

Comment:
You need to add the deprecated function back.

Reply:
Since this inherits from `Triangulation` we don't need to add a shim here - just in the base class.